src/lread.c

```diff
@@ -4009,6 +4009,20 @@ check_obarray (Lisp_Object obarray)
   return obarray;
 }
 
+/* Like check_obarray, but for the global Vobarray. */
```

Comment:
No, it's not exposed in any .h files. I got a linker error when I tried. Is there another way?